spec/jobs/multi_region_kms_migration/profile_migration_job_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe MultiRegionKmsMigration::ProfileMigrationJob do
+  let!(:profiles) { create_list(:profile, 4, :with_pii) }
+  let!(:single_region_ciphertext_profiles) do
+    single_region_profiles = profiles[2..3]
+    single_region_profiles.each do |profile|
+      profile.update!(
+        encrypted_pii_multi_region: nil,
+        encrypted_pii_recovery_multi_region: nil,
+      )
+    end
+    single_region_profiles
+  end
+  let!(:multi_region_ciphertext_profiles) { profiles[0..1] }
+
+  describe '#perform' do
+    it 'does not modify records that do have multi-region ciphertexts' do
+      profile = multi_region_ciphertext_profiles.first
```

Comment:
Hmm, we could but I don't super love mocking `update`. There are lots of ways to update the attributes on a record. For example, in this case `expect(profile).not_to receive(:update)` would not work because we call `update!`. I much prefer looking at the actual source of truth. I think this plus the spec to ensure we only retrieve the updatable profiles and the tests on the migrator should have us covered.